<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BasicDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="bnOpen">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bnClose">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭设备</string>
    </property>
   </widget>
   <widget class="QLabel" name="lExposure">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>140</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>曝光</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbExposure">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>140</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lGain">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>增益</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbGain">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>170</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lFrameRate">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>帧率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbFrameRate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>200</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bnGetParam">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>获取参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bnSetParam">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置参数</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>751</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbDisplayFrame">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>330</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="PixelFormatComboBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bnInputOneFrame">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>270</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>获取视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bnStopRecord">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>270</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止录像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bnInputVideo">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>300</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导入视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bnGetOneFrame">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>获取一张图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bnSaveFrame">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbShowFrame">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>610</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>bnOpen</tabstop>
  <tabstop>bnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
